control-plane/connect-inject/controllers/endpoints/endpoints_controller_test.go

```diff
@@ -4821,7 +4822,7 @@ func TestCreateServiceRegistrations_withTransparentProxy(t *testing.T) {
 					LivenessProbe: &corev1.Probe{
 						Handler: corev1.Handler{
 							HTTPGet: &corev1.HTTPGetAction{
-								Port: intstr.FromInt(exposedPathsLivenessPortsRangeStart),
+								Port: intstr.FromInt(20300),
```

Comment:
Should this maybe be a constant since I see 20300 being used in at least 2 places?  endpoints_controller_test and mesh_webhook ?

Reply:
Sounds good!